src/components/Quiz/QuizzesStats.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useContext } from "react"
+import {
+  Box,
+  Circle,
+  Flex,
+  Grid,
+  GridItem,
+  Icon,
+  Progress,
+  Stack,
+  Text,
+} from "@chakra-ui/react"
+import { FaTwitter } from "react-icons/fa"
+
+import Button from "../Button"
+import Translation from "../Translation"
+import { TrophyIcon } from "../icons/quiz"
+
+import { QuizzesHubContext } from "./context"
+
+import { getNumberOfCompletedQuizzes, getTotalQuizzesPoints } from "./utils"
+
+import { QuizShareStats } from "../../types"
+
+import { ethereumBasicsQuizzes, usingEthereumQuizzes } from "../../data/quizzes"
+
+// TODO: track event on matomo
+const shareOnTwitter = ({ score, total }: QuizShareStats): void => {
```

Comment:
Should we have a different message if score === 0 / they haven't taken any quizzes? @konopkja 

<img width="588" alt="Screenshot 2023-05-29 at 09 01 57" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/3c50313a-d903-45b2-a300-ed4530f15c80">

Reply:
Absolutely, @nhsz  is that possible? Something like: 

_How well do you know Etheruem? Check out these Ethereum quizzes on Ethereum.org: ethereum.org/quizzess @ethereumquiz #ethereum #quiz_

Reply:
Done